src/series/column.js

```diff
@@ -0,0 +1,120 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.series.column = function() {
+
+        var decorate = fc.util.fn.noop,
+            xScale = d3.time.scale(),
+            yScale = d3.scale.linear(),
+            x1Value = function(d, i) { return d.close; },
+            yValue = function(d, i) { return d.date; },
+            x0Value = d3.functor(0),
+            columnWidth = fc.util.fractionalBarWidth(0.75);
+
+        var dataJoin = fc.util.dataJoin()
+            .selector('g.column')
+            .element('g')
+            .attr('class', 'column');
+
+        var yValueScaled = function(d, i) { return yScale(yValue(d, i)); };
+
+        var column = function(selection) {
+            selection.each(function(data, index) {
+
+                var filteredData = data.filter(function(d, i) {
+                    return x0Value(d, i) !== undefined &&
+                        x1Value(d, i) !== undefined &&
+                        yValue(d, i) !== undefined;
+                });
+
+                var g = dataJoin(this, filteredData);
+
+                var height = columnWidth(filteredData.map(yValueScaled));
+
+                var pathGenerator = fc.svg.bar()
+                    .x(0)
+                    .y(0)
+                    .height(height)
+                    .horizontalAlign('right')
+                    .width(0);
+
+                var y = function(d, i) { return yValueScaled(d, i); },
+                    x1 = function(d, i) { return xScale(x1Value(d, i)); },
+                    x0 = function(d, i) { return xScale(x0Value(d, i)); };
+
+                g.enter()
+                    .attr('transform', function(d, i) {
+                        return 'translate(' + x0(d, i) + ', ' + y(d, i) + ')';
+                    })
+                    .append('path')
+                    .attr('d', function(d) { return pathGenerator([d]); });
```

Comment:
They are on both because when the bar is rendered via the enter selection it has a zero height. This means that if it is used with a transition it will grow upwards from y0 to y1.

It's a little unusual in that our other series types do not consider transitions - but I think it is a neat idea. Creating decent transitions with D3 takes a bit of thought.

Might be worth adding a quick comment as this behaviour probably isn't that obvious from the code.